#!/usr/bin/env python

PACKAGE = "throttle"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("THROTTLE_FORWARD_PWM", int_t, 0, "Parameter value to go straight", 0, 0, 1500)
gen.add("THROTTLE_STOPPED_PWM", int_t, 0, "Parameter value to stop", 0, 0, 1500)
gen.add("THROTTLE_REVERSE_PWM", int_t, 0, "Parameter value to go back", 0, 0, 1500)


exit(gen.generate(PACKAGE, "throttle_config", "Throttle"))